### ISSUE TITLE
'StethoHttpClientResponse' is missing implementations for these members: - HttpClientResponse.compressionState in flutter_stetho

### ISSUE THREAD

**Issue by Sachinsowri** [brianegan/flutter_stetho#18]
I am tried to use flutter_stetho package when it Complie the complier shown ..

Launching lib/main.dart on Android SDK built for x86 in debug mode...

Compiler message: file:///home/administrator/softwares/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_stetho-0.2.2/lib/src/http_client_response.dart:4:7: Error: The non-abstract class 'StethoHttpClientResponse' is missing implementations for these members: - HttpClientResponse.compressionState Try to either - provide an implementation, - inherit an implementation from a superclass or mixin, - mark the class as abstract, or - provide a 'noSuchMethod' implementation.

class StethoHttpClientResponse extends StreamView> ^^^^^^^^^^^^^^^^^^^^^^^^ org-dartlang-sdk:///third_party/dart/sdk/lib/_http/http.dart:1967:42: Context: 'HttpClientResponse.compressionState' is defined here. HttpClientResponseCompressionState get compressionState; ^^^^^^^^^^^^^^^^ Compiler failed on /home/administrator/projects/mobile/sampleapp/lib/main.dart

FAILURE: Build failed with an exception. * Where: Script '/home/administrator/softwares/flutter/packages/flutter_tools/gradle/flutter.gradle' line: 665

What went wrong: Execution failed for task ':app:compileflutterBuildDebugX86'.
Process 'command '/home/administrator/softwares/flutter/bin/flutter'' finished with non-zero exit value 1

Try: Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

Get more help at https://help.gradle.org

BUILD FAILED in 20s Gradle task assembleDebug failed with exit code 1 Exited (sigterm)

**Comment by JunusErgin** [comment]
I'm having the sam problem. 
Flutter: 1.7.8.
I'm using Dart 2.4.0; 
Windows 10; Android Studio.

Related: https://github.com/flutter/flutter/issues/34922

**Comment by vanelizarov** [comment]
Fixed in PR #24

**Comment by ariefannur** [comment]
just use git url for dependencies
```
flutter_stetho:
    git:
      url: https://github.com/brianegan/flutter_stetho.git
```

**Comment by brianegan** [comment]
Just published 0.3.0 after a bit of testing to make sure all was good! Please try the new version and let me know if you have any further issues.
